### PR TITLE
[DOCS] Add Webhook connector to case and connector docs #2221

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Addresses https://github.com/elastic/security-docs/issues/2221.

A couple of notes about this PR:
- To align the Security docs with the changes @lcawl has made in https://github.com/elastic/observability-docs/pull/2083, I also removed the table that contains configuration details for each connector and then linked to the appropriate topics in the Kibana docs where connector config details are maintained. 
-  ~I've commented out the link to the new Kibana reference topic for the Webhook connector since the PR for those docs (https://github.com/elastic/kibana/pull/137726) hasn't been merged yet. Once the PR is merged, I'll uncomment the reference.~

Previews:
- [Cases](https://security-docs_2297.docs-preview.app.elstc.co/guide/en/security/master/cases-overview.html)
- [Configure external connections](https://security-docs_2297.docs-preview.app.elstc.co/guide/en/security/master/cases-ui-integrations.html)

### CODE REVIEW

**Review by lcawl — APPROVED** [review]
LGTM!

File: docs/cases/cases-ui-integrations.asciidoc [hunk 1]
```diff
@@ -24,140 +25,8 @@ https://www.elastic.co/subscriptions[appropriate license], and your role needs *
...
-overwritten.
-
-| *Description* | Mapped to the {swimlane} `Description` field. When an update to a case description is sent to {swimlane}, the field that is mapped to the {swimlane} `Description` field is overwritten.
-
-| *Comments* | Mapped to the {swimlane} `Comments` field. When a new comment is added to a case, or an existing one is updated, the field that is mapped to the {swimlane} `Comment` field is appended. Comments are posted to the {swimlane} incident record individually.
-
-|===
+. Select the system to send cases to: *{sn}*, *{jira}*, *{ibm-r}*, *{swimlane}*, or *{webhook-cm}*.
+. Enter your required settings. For connector configuration details, refer to {kibana-ref}/servicenow-action-type.html[{sn-itsm} connector], {kibana-ref}/servicenow-sir-action-type.html[{sn-sir} connector], {kibana-ref}/jira-action-type.html[{jira} connector], {kibana-ref}/resilient-action-type.html[{ibm-r} connector], {kibana-ref}/swimlane-action-type.html[{swimlane} connector], or {kibana-ref}/cases-webhook-action-type.html[{webhook-cm} connector].
```
Comment (joepeeples) [inline docs/cases/cases-ui-integrations.asciidoc:29]: A bullet list might be tidier for a long list of links:
```suggestion
. Enter your required settings. For connector configuration details, refer to:
- {kibana-ref}/servicenow-action-type.html[{sn-itsm} connector]
- {kibana-ref}/servicenow-sir-action-type.html[{sn-sir} connector]
- {kibana-ref}/jira-action-type.html[{jira} connector]
- {kibana-ref}/resilient-action-type.html[{ibm-r} connector]
- {kibana-ref}/swimlane-action-type.html[{swimlane} connector]
- {kibana-ref}/cases-webhook-action-type.html[{webhook-cm} connector]
```

**Review by joepeeples — APPROVED** [review]
LGTM! Left one totally optional suggestion.

**Comment by karanbirsingh-qasource** [comment]
Hi @jmikell821 

we have reviewed the shared preview for Update Host Risk Score Setup process and all the required change are present and correct. ✔️ 

image

image


Hence we are adding "QA:Validated" tag to it.


thanks !!

**Review by jmikell821 — APPROVED** [review]
The bulleted list looks much cleaner. LGTM! 👍

**Review by benironside — APPROVED** [review]
LGTM

**Pull request merged**